site/src/components/pages/hub/BlockDataContainer.tsx

```diff
@@ -29,15 +37,23 @@ export const BlockDataContainer: VoidFunctionComponent<
   BlockDataContainerProps
 > = ({ metadata, schema, blockModule }) => {
   const [blockDataTab, setBlockDataTab] = useState(0);
-  const [blockTab, setBlockTab] = useState(0);
+  const [blockVariantsTab, setBlockVariantsTab] = useState(0);
   const [blockModalOpen, setBlockModalOpen] = useState(false);
+  const [alertSnackBarOpen, setAlertSnackBarOpen] = useState(false);
 
-  const example = metadata.examples?.[0];
+  const example = {
```

Comment:
Form the initial example as a combination of the default example, the example of the first variant, and the variant's properties.

The priority of assignment is in ascending order, ie, downwards.